Podfile

```diff
@@ -218,8 +218,8 @@ abstract_target 'Apps' do
 
     pod 'Gridicons', '~> 1.1.0'
 
-    pod 'WordPressAuthenticator', '~> 1.43.1'
-    # pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => ''
+    # pod 'WordPressAuthenticator', '~> 1.43.1'
+    pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => 'remove-lottie'
```

Comment:
Of course, before merging, we'll want at least want https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/640 to be merged and shipped as a bet or stable.

Reply:
Added `[Status] DO NOT MERGE]` to the PR to make that even more visible.

Reply:
Nice. Better than just a comment 😅 Plus gets Peril to help us with it, too.